### ISSUE TITLE
Vite Configuration Incomplete

### ISSUE THREAD

**Issue by EntranceJew** [samchon/typia#259]
I'm running a project based out of [vitesse-electron](https://github.com/adensW/vitesse-electron) and I set typescript-json it up as best I could by applying the relevant configs under `main/tsconfig.ts` and `main/vite.config.ts` but npm run watch still gives me:
```
Error: Error on TSON.stringify(): no transform has been configured. Configure the "tsconfig.json" file following the [README.md#setup](https://github.com/samchon/typescript-json#setup)
```
`TSON.stringify<MyClass>()` throws an error still.

I cannot find a way for a script inside the `main/src/index.ts` to run properly following only the instructions in the readme.
if there are more steps involving ttsc other than those outlined I will need info on them, because otherwise I would expect the `npm run watch` from this repository to work.

**Comment by samchon** [comment]
Can you show me your repo or a new repo demonstrating that bug?

**Comment by WelcomeDev** [comment]
Don't really know where to put this question. Set up for `vite` works kind of fine. Though there's one real issue. Hot reload is always "one step late" (just try smth to change). 

The problem isn't really in "typescript-json".  The bag guy here is "@rollup/plugin-typescript". It just breaks HMR. But maybe you have some kind of workaround? I've tried other similar package - "rollup-plugin-typescript-2". But it doesn't work at all. 
And I'm out of ideas how to setup runtime validation with `vite`. Your answer would be much appreciated.

Demo [Stackblitz](https://stackblitz.com/edit/vitejs-vite-phea3a?file=src%2FApp.tsx)

**Comment by samchon** [comment]
How about reporting the bug into `vite` repo?

Until be fixed, how about compiling TSON in other build environment and your main project to import it.

**Comment by WelcomeDev** [comment]
Thanks for your fast response! I've created the issue at [vite](https://github.com/vitejs/vite/issues/10952). And already got the response. At first place I've messaged you, because you pointed vite with `@rollup/plugin-typescript` in multiple answers as the best solution. So, I assumed you, probably, know some workaround.

But it seems, it's not really possible for now. Probably it's worth pointing the issue at Readme configuration section. I'll try to follow [patak-dev](https://github.com/patak-dev) answer and fork esbuild. But it seems to be the nag.

All this mess is for API runtime validation at React frontend project. And I haven't found workaround with other bundlers yet. Vite was the "nearest to the best" solution
